.github/workflows/release-drafter.yml

```diff
@@ -2,12 +2,12 @@ name: Release Drafter
 
 on:
   push:
-    branches: ['master']
+    branches: ['main']
```

Comment:
Since this is a new branch, please remove the old `master` and adjust `gh-pages` if necessary.